config/locales/fr.yml

```diff
@@ -1,30 +1,30 @@
 fr:
   edit: Modification
-  failure_when_forbidden:
+  failure_when_forbidden: Veuillez vérifier l'URL ou réessayer.
   feed_latest: RubyGems.org | Derniers Gems
   feed_subscribed: RubyGems.org | Gems abonnés
   footer_about: RubyGems.org est le service d&rsquo;hébergement de la communauté Ruby. Publiez vos gems instantanément et utilisez-les. Utilisez l'API pour interagir et trouver des informations sur les gems disponibles. Contribuez et améliorez ce site avec nous !
   form_disable_with: Veuillez patienter...
   help_url: http://help.rubygems.org
   locale_name: Français
   none:
-  not_found:
+  not_found: Introuvable
   please_sign_up: Accès refusé. Inscrivez-vous sur https://rubygems.org
   sign_in: Connexion
   sign_up: Inscription
-  multifactor_authentication:
+  multifactor_authentication: Authentication Multifacteur
```

Comment:
"Authentification" rather than "Authentication"?

Reply:
Oh yes ! Good catch, I will change it everywhere